scls-format/src/Cardano/SCLS/Internal/Serializer/External/Impl.hs

```diff
@@ -42,72 +52,185 @@ serialize ::
   NetworkId ->
   -- | Slot of the current transaction
   SlotNo ->
-  -- | Namespace for the data entries
-  Text ->
   -- | Input stream of entries to serialize, can be unsorted
-  Stream (Of a) IO () ->
+  S.Stream (S.Of (Text, S.Stream (S.Of a) IO ())) IO () ->
```

Comment:
Maybe define a type alias for this, since it's used multiple times. Might also help in distinguishing it from other similar but not quite the same type signatures I found in the rest of the code.